x/programs/cmd/simulator/vm/config/config.go

```diff
@@ -87,17 +86,14 @@ func New(nodeID ids.NodeID, b []byte) (*Config, error) {
 func (c *Config) setDefault() {
 	c.LogLevel = c.Config.GetLogLevel()
 	c.MempoolSize = c.Config.GetMempoolSize()
-	c.MempoolPayerSize = c.Config.GetMempoolPayerSize()
 	c.StateSyncServerDelay = c.Config.GetStateSyncServerDelay()
 	c.StreamingBacklogSize = c.Config.GetStreamingBacklogSize()
-	c.VerifySignatures = c.Config.GetVerifySignatures()
```

Comment:
Why remove this one?

Reply:
it wasn't being used as far as I could tell.

Reply:
more specifically https://github.com/ava-labs/hypersdk/blob/main/vm/dependencies.go#L28 doesn't seem to have GetVerifySignatures() anymore.

Reply:
It is now `GetVerifyAuth`: https://github.com/ava-labs/hypersdk/blob/0bd179ff03e4cc7672faaa69502d06a73d1d9fb5/vm/dependencies.go#L32

Reply:
I'll update this rq